src/markdown-pages/automate-workflows/automated-tagging.mdx

```diff
@@ -0,0 +1,313 @@
+
+
+## Before you begin
+To run this example there are several pre-requisite steps, most of which involve setting up the right configurations, and at least for now, building a [docker](https://docs.docker.com/get-docker/) image. 
+
+We use several of our open source projects to make this all work. Let's take a look at them now.
+* [Demo Deployer](https://github.com/newrelic/demo-deployer) - This is the engine that drives all the automation. For convenience, you'll clone this repo and build a docker image. In future iterations, we'll publish the image in docker hub or someplace similar. 
+* [Demo Nodetron] (https://github.com/newrelic/demo-nodetron) - This is the simple app that will be provisioned on a host. It will give you a simple web page and an API that you can hit with a tool like [Postman](https://www.postman.com/downloads/) depending on how much traffic you want to generate. 
+* [Demo New Relic Instrumentation](https://github.com/newrelic/demo-newrelic-instrumentation) - This is the project that we use to layer in the instrumentation for the infrastructure and the application. 
+
+We kept each project distinct to encapsulate its behavior. The idea being that we can simply add more behaviors and then weave them in to assemble more complex scenarios. Cool, right?
+
+<Steps>
+
+<Step>
+
+### Build the Deployer
+Let's get the deployer built. Essentially, this is cloning the repo and building the docker image. If you don't have docker installed, you can download it [here](https://docs.docker.com/get-docker/). 
+
+* Clone the deployer repo
+
+```bash
+git clone git@github.com:newrelic/demo-deployer.git $HOME/deployer
+```
+
+* Build the docker image
+
+```bash
+docker build -t deployer .
+```
+</Step>
+
+<Step>
+
+### Create Config files
+In this example, we will actually provision a host on either AWS or Asure, depending on which one you choose. The deployer that you just built, uses a configuration file that has references your creds so it can invoke the necessary APIs. The instructions to create your configuraiton file are [here](https://github.com/newrelic/demo-deployer/blob/main/documentation/user_config/README.md). In the end, you will have something that looks similar to the file on the right. 
+
+```json
+{
+  "credentials": {
+
+    "aws": {
+      "apiKey": "my_aws_api_key",
+      "secretKey": "my_aws_secret_key",
+      "secretKeyPath": "/path/to/my/secretkey.pem",
+      "region": "my_aws_region"
+    },
+
+    "azure": {
+      "client_id": "my_client_id",
+      "tenant": "my_tenant_id",
+      "subscription_id": "my_subscription_id",
+      "secret": "my_secret",
+      "region": "my_region",
+      "sshPublicKeyPath": "/path/to/my/id_rsa.pub"
+    },
+
+    "newrelic": {
+      "licenseKey": "my_new_relic_license_key",
+      "insightsInsertApiKey": "my_new_relic_insights_api_key",
+      "accountId": "my_new_relic_account_id"
+    },
+
+    "git": {
```

Comment:
We can omit the whole "git" section, this is no longer useful now with public repo.